packages/shared/src/components/sources/SourceActions/SourceActionsWithNotifyExperiment.tsx

```diff
@@ -44,9 +48,11 @@ export const SourceActionsWithNotifyExperiment = ({
     source,
   });
 
+  const [isNotifyOpen, toggleNotifyOpen] = useToggle(false);
```

Comment:
We should not need the state anymore and just use the existing hook to enable PNs 🥳